### DISCUSSION TITLE
v0.1.0 pre-release - The base path is "/device-identifier/v0.1.0" instead of "/device-identifier/v0.1". Will this base path remain in the next release?

### DISCUSSION THREAD

**Discussion by trehman-gsma** [camaraproject/DeviceIdentifier#65] [General]
Hi @eric-murray @Kevsy 

In [v0.1.0](https://github.com/camaraproject/DeviceIdentifier/releases/tag/device-identifier-0.1.0), the base path is [/device-identifier/v0.1.0](https://github.com/camaraproject/DeviceIdentifier/blob/device-identifier-0.1.0/code/API_definitions/CAMARA%20Mobile%20Device%20Identifier%20API.yaml#L113). The latest [CAMARA guidance](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md?plain=1#L586) seems to be `/v0.1` instead of `/v0.1.0`. Will the `/device-identifier/v0.1.0` path remain in future versions?

Also - I see that [v0.1.0](https://github.com/camaraproject/DeviceIdentifier/releases/tag/device-identifier-0.1.0) is labelled as a pre-release. Does this mean that there will be a v0.1.0 official release where potentially the base path will be changed?

**Comment by eric-murray** [comment]
Hi @trehman-gsma

I'm currently working my way through the latest CAMARA guidance to see what changes are required for the next version. That will include the basepath. The current (pre-)release would be an "alpha" release under the current guidelines, but was released before these were complete, so I just went straight to an "initial" release.

I'm slowly trying to turn the wip into what will become the release candidate for the next version. As the current release is 0.1.0 with no qualification, the release candidate will need to be 0.2.0-rc.1, leading to 0.2.0 at some point. I think having a separate "pre-release" and "release" with the same v0.1.0 numbering will cause confusion.

My understanding was that, for "initial" (i.e. v0) releases, it did not matter if it was formally a github release or pre-release, so I chose "pre-release" for 0.1.0 to be safe. When we get to releasing 0.2.0, I'll confirm the latest guidance on this.

**Comment by trehman-gsma** [comment]
Thanks @eric-murray.

We are in contact with an operator implementing v0.1.0. We will advise them to use `/device-identifier/v0.1.0` as that is what the YAML specifies but also to be aware that the base path may change in future releases subject to CAMARA guidelines. 

Closing this discussion, thanks.
